Explain how to set envars for troubleshooting (#573)
grafana/docker-otel-lgtm: README.md

### Enable logging

> You can enable logging in the .env file for troubleshooting:

Reviewer: Given that the variables can be provided through multiple methods—not just an .env file, e.g. set within the compose file — I recommend a more generic formulation:

```suggestion
You can enable logging for troubleshooting via environment variables:
```
